src/main/java/seedu/realodex/logic/parser/Prefix.java

```diff
@@ -1,30 +1,91 @@
 package seedu.realodex.logic.parser;
 
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
 /**
  * A prefix that marks the beginning of an argument in an arguments string.
  * E.g. 't/' in 'add James t/ friend'.
  */
 public class Prefix {
     private final String prefix;
+    private final String prefixRepresentation;
```

Comment:
perhaps change "prefixRepresentation" to something more intuitive? like prefixDescription? idk

Reply:
Changed thanks!